components/service/glean/src/main/java/mozilla/components/service/glean/storages/StringSetsStorageEngine.kt

```diff
@@ -14,30 +14,30 @@ import mozilla.components.support.ktx.android.org.json.toList
 import org.json.JSONArray
 
 /**
- * This singleton handles the in-memory storage logic for string lists. It is meant to be used by
- * the Specific String List API and the ping assembling objects.
+ * This singleton handles the in-memory storage logic for string sets. It is meant to be used by
+ * the Specific String Set API and the ping assembling objects.
  *
  * This class contains a reference to the Android application Context. While the IDE warns
  * us that this could leak, the application context lives as long as the application and this
  * object. For this reason, we should be safe to suppress the IDE warning.
  */
 @SuppressLint("StaticFieldLeak")
-internal object StringListsStorageEngine : StringListsStorageEngineImplementation()
+internal object StringSetsStorageEngine : StringSetsStorageEngineImplementation()
 
-internal open class StringListsStorageEngineImplementation(
-    override val logger: Logger = Logger("glean/StringsListsStorageEngine")
-) : GenericStorageEngine<List<String>>() {
+internal open class StringSetsStorageEngineImplementation(
+    override val logger: Logger = Logger("glean/StringsSetsStorageEngine")
+) : GenericStorageEngine<Set<String>>() {
     companion object {
-        // Maximum length of any list
-        const val MAX_LIST_LENGTH_VALUE = 20
-        // Maximum length of any string in the list
+        // Maximum length of any set
+        const val MAX_SET_LENGTH_VALUE = 20
```

Comment:
nit: `LENGTH` -> `SIZE` (to match the Kotlin API for the underlying data structure...?)